developer-docs/bolt_server.md

```diff
@@ -47,11 +109,79 @@ The Target be a JSON object. The following keys are available:
 - `host-key-check`: Bool, *optional* - Whether to perform host key validation when connecting over SSH (Default: `true`).
 - `sudo-password`: String, *optional* - Password to use when changing users via `run-as`
 
+### WinRM Target Object
+The Target be a JSON object. The following keys are available:
+- `hostname`: String, *required* - Target identifier.
+- `password`: String, *optional* - Password for WinRM transport authentication.
+- `port`: Integer, *optional* - Connection port (Default: `5986`, or `5985` if `ssl: false`.)
+- `user`: String, *optional* - Login user (Default: `root`).
+- `connect-timeout`: Integer, *optional* - How long Bolt should wait when establishing connections.
+- `tmpdir`: String, *optional* - The directory to upload and execute temporary files on the target.
+- `ssl`: Boolean, *optional* - When true, Bolt will use normal http connections for WinRM (Default: `true`).
```

Comment:
Doesn't it use https when true and http when false?

Reply:
@katelopresti I copied this from https://puppet.com/docs/bolt/0.x/bolt_configuration_options.html#winrm-transport-configuration-options looks like we need to update it?